tests/xharness/Jenkins/TestTasks/TestTask.cs

```diff
@@ -32,6 +33,16 @@ public abstract class TestTask
 		public bool BuildOnly;
 		public string KnownFailure;
 
+		public TestTask ()
+		{
+			// no, Interlocked.Increment is not going to do the samething, it is a two step, set and increment, 
+			// while Interlocked.Increment is just atomic for the increment. You might think that Interlocked.Ingrement 
```

Comment:
Don't spam a new build for this but typo! 😁 

Definitely `Interlocked.Ingrement` would not work but `Increment` may.

Reply:
I'm going to spam builds because I'm going to  implement what @rolfbjarne suggested and see what happens ;)